### DISCUSSION TITLE
Am I missing something? App router tutorial strange behavior

### DISCUSSION THREAD

**Discussion by vladutclp** [vercel/next.js#64819] [App Router] [answered]
Hi everyone, I'm following the next js dashboard tutorial and my app is behaving in a strange way.

I have the following folder structure **app/ui/dashboard,** with a **page.tsx** file insinde the **dashboard** folder. I would expect that going to **http://localhost:3001/dashboard** would render the dashboard page component.

But it returns me 404. It works though like this http://localhost:3001/ui/dashboard but the **ui** folder does not have any **page.tsx** file.

Am I doing something wrong?

Thank you!

![image](https://github.com/vercel/next.js/assets/22237268/a8bc04a7-8cbf-46a2-b9ea-7fa59b0ae0a7)
![image](https://github.com/vercel/next.js/assets/22237268/f80a113b-6241-40a7-a9e5-cffe02f4b3f4)
![image](https://github.com/vercel/next.js/assets/22237268/3bc0583b-333f-4ae3-a5de-bbd060d1f75a)

**Comment by sensorario** [comment, marked as answer]
Paths start form app:

App: /
App/ui: /ui
App/ui/dashboard: /ui/dashboard

**Reply by vladutclp** [reply]
But in the tutorial this is not specified: 
![image](https://github.com/vercel/next.js/assets/22237268/2f0feafa-74a1-4084-a087-a9bda8ba4a28)

https://nextjs.org/learn/dashboard-app/creating-layouts-and-pages#creating-the-dashboard-page

**Reply by vladutclp** [reply]
And also based on this https://nextjs.org/learn/dashboard-app/creating-layouts-and-pages#creating-the-dashboard-page, it should only work for folders that have **page.tsx/jsx** inside of them: 
![image](https://github.com/vercel/next.js/assets/22237268/3714030d-c8b3-4839-9ff2-73e0e8c67b9a)

**Comment by sensorario** [comment]
Ok, ... try witj /ui/dashboard, check that everything is working (I've did it) and open an issue in the documentation. If a segment does not contain page.tsx is... "private". app/foo/bar/ciccio/pasticcio/page.tsx means /foo/bar/ciccio/pasticcio even if /foo, bar/, ... are not containing page.tsx

**Comment by vladutclp** [comment]
I figured out what was the problem. I forgot to create the **dashboard** folder in the **app** and create the page there.

I will close this discussion. @sensorario thanks for your answer :)
